### ISSUE TITLE
How do I know if it saved my session?

### ISSUE THREAD

**Issue by rajeshisnepali** [tmux-plugins/tmux-resurrect#344]
There was no message that it saved my session.

**Comment by sqqqrly** [comment]
On mine, I see "Saving..." followed by "TMUX session saved".

If I look in ~/.tmux/resurrect, I see the resurrect text files.   The last one has a soft link named "last" pointing to it.

**Comment by rajeshisnepali** [comment]
Yes, you're right. Later, after the issue, I can also see "Saving...." at the bottom.

But I think I got something else here. Would you like to see this issue as well? [https://github.com/tmux-plugins/tmux-resurrect/issues/347](https://github.com/tmux-plugins/tmux-resurrect/issues/347)
